addons/payment/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+unlink_by_xmlid = [
+    'payment.payment_acquirer_custom'
+]
+
+
+def map_payment_acquirer_state(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE payment_acquirer
+        SET state = CASE WHEN {website_published} = TRUE THEN 'enabled'
+            ELSE 'disabled' END
+        WHERE state = 'prod'""".format(
+            website_published=openupgrade.get_legacy_name('website_published')
+        )
+    )
+
+
+def update_account_invoice_transaction_rel_table(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE account_invoice_transaction_rel aitr
+        SET invoice_id = am.id
+        FROM account_move am
+        WHERE am.old_invoice_id = aitr.old_invoice_id
```

Comment:
@MiquelRForgeFlow You have approved this, but I get error so maybe I am doing something wrong.  THere appears to be no column `aitr.old_invoice_id` but rather just `aitr.invoice_id`. It would make sense looking at query but maybe not.